CHANGELOG.md

```diff
@@ -28,7 +32,103 @@ The Tidepool platform API
+* Update structured store to allow optional update operators and query modifiers
+* Add unstructured store DeleteDirectory to provide batch delete operations
+* Add unstructured store Options during put operation to allow setting content type on S3 object
+* Remove context from unstructured store test mocks as it can change unrelated to target under test
+* Update aws package and interfaces to enable batch object delete
+* Update pagination to not include page or size if defaults are specified
+* Update log package to enable easier addition of contextual data into context and logger
+* Update structure parser to allow reporting errors in contained data by reference
+* Add Makefile targets to execute unit tests until failure
+* Fix import alias for any import path ending in /v1
+* Update dump users tool to be more generic and usable
+* Return HTTP status code 413 Request Entity Too Large when blob size exceeds maximum
+* Replace blob.Create with blob.Content
+* Remove blob Content-Length header usage and blob.Content.Size as it does not work with content encoding
+* Update test config reporter for better debugging
+* Fix randomly failing profile mongo test
+* Fix unused tests
+* Fix import ordering due to Golang tool changes
+* Update dependencies
+* Update to Go 1.11.1
+* Add user id to ArchiveDeviceDataUsingHashesFromDataSet selector to properly use indexes
+* Fix data set id validation to allow Jellyfish id format
+
+## v1.30.1
+
+* Relax Dexcom API validations to match Tidepool data model validations
+* Rename insulin dose constants to properly include units
+
+## v1.30.0
+
+* Fix Dexcom API v2 edge cases
+* Handle unexpected data from Dexcom API v2
+* Update to Dexcom API v2
+* Capture cgm settings for changed Dexcom API devices
+* Capture new fields from Dexcom API v2 data
+* Pull data from Dexcom API in 30 day increments rather than 90
+* Add Dexcom time struct for proper formatting during serialization
+* Use constants consistently throughout dexcom package
+* Remove device info from Dexcom API data as not useful
+* Remove unnecessary TODO for Dexcom API mmol/L support
+* Update cgm settings to allow for new settings from Dexcom API
+* Add default alert settings and alternative scheduled alert settings to cgm settings
+* Add test helpers for data parser serialization
+* Add pointer.Equal functions
+* Deprecate current cgm settings alert structures
+* Update oauth/client package tests to latest best practices
+* Update platform package tests to latest best practices
+* Update client package tests to latest best practices
+* Update Dexcom types to latest best practices
+* Add data store benchmark tool
+* Rename local variable name context to ctx for consistency
+* Remove unused code in data store mongo implementation
+* Update tool dependencies
+* Add build tools as additional dependencies
+* Update Makefile to install build tools from vendor directory
+* Update Makefile to remove unused tools
+
+## v1.29.0
```

Comment:
Do we need the full Tidepool ChangeLog ?

Reply:
I don't know... Maybe something to discuss in a daily ?
Having it may be easier in the future merge.